docs/user_documentation/guide-questionnaire.md

```diff
@@ -37,4 +37,28 @@ Go to the data explorer and look for the (in our case) Simple Questionnaire tabl
+| id                 | demo_report_header   | string                 | TRUE        | TRUE           |              |                    |                                                 |                 |
+| intro-en           | demo_report_header   | text                   | FALSE       | FALSE          |              |                    |                                                 |                 |
+| intro-nl           | demo_report_header   | text                   | FALSE       | FALSE          |              |                    |                                                 |                 |
+| intro-fr           | demo_report_header   | text                   | FALSE       | FALSE          |              |                    |                                                 |                 |
+| intro-de           | demo_report_header   | text                   | FALSE       | FALSE          |              |                    |                                                 |                 |
+| intro-es           | demo_report_header   | text                   | FALSE       | FALSE          |              |                    |                                                 |                 |
+| intro-it           | demo_report_header   | text                   | FALSE       | FALSE          |              |                    |                                                 |                 |
+| intro-pt           | demo_report_header   | text                   | FALSE       | FALSE          |              |                    |                                                 |                 |
+| logo               | demo_report_header   | text                   | FALSE       | FALSE          |              |                    |                                                 |                 |
+| id                 | demo_questionnaire   | string                 | AUTO        | FALSE          |              |                    |                                                 |                 |
+| report_header      | demo_questionnaire   | xref                   | FALSE       | FALSE          |              | demo_report_header |                                                 |                 |
+| questions          | demo_questionnaire   | compound               | FALSE       | FALSE          |              |                    |                                                 |                 |
+| question1          | demo_questionnaire   | bool                   | FALSE       | FALSE          |              |                    | Do you think this questionnaire is quite small? | questions       |
+
+The intro should be some text information about the PDF report. The logo should be a data URL (and therefore is of type text). 
```

Comment:
Have you considered putting the logo in the file store instead and creating an xref to FileMeta. We have a controller that'll serve the file then.

Reply:
> The lib used to create the pdf expects a dataUrl ( do we serve that i don't know ? )

https://my.molgenis.org/files/my_file_meta_entity

Disadvantages of using a URL:
- Browser security warnings when serving MOLGENIS via https and image via http
- No way to use report logo when running MOLGENIS in an environment without internet connection (e.g. a Lifelines workspace)

Reply:
Disregard my previous comment since the logo value is not a URL but a base64 encoded string